cmd/puma-dev/command_test.go

```diff
@@ -100,3 +100,7 @@ func TestCommand_link_reassignExistingApp(t *testing.T) {
 
 	assert.Equal(t, fmt.Sprintf("! App '%s' already exists, pointed at '%s'\n", appAlias, appDir1), actual2)
 }
+
+func TestCommand_status(t *testing.T) {
+	assert.Nil(t, status())
```

Comment:
You can make this a bit stronger by utilizing the `WithStdOutCaptured` helper in the `devtest` package and assert that this simple case prints the correct error message.

Reply:
and that it reads the port information correctly.

Reply:
thanks! I am admittedly quite new to writing Go tests...

Reply:
of course! admittedly, so am i. but, i've been slowly working on adding enough sugar in here to make testing this stuff a bit easier. 

take a look here for an example of how to use that helper i mentioned. i'm happy to answer questions / address feedback.

https://github.com/puma/puma-dev/blob/69b57944a9a27cc6df94d01d55d578e03a35e72b/cmd/puma-dev/command_test.go#L85-L92